utils/README.md

```diff
@@ -0,0 +1,41 @@
+# Equivalence Testing
+
+Equivalence testing is to evaluate the similarity of image or audio results. 
+Before conducting testing, please follow the instructions below to configure the virtual environment.
+
+## Configure Virtual Environment
+
+### Create Virtual Environment
+
+```
+$ python -m venv EquivalenceTest.venv
+```
+
+### Managing Packages with pip
+
+```
+$ pip install pillow
+$ pip install numpy
+$ pip install sys
```

Comment:
Can you put this in a `requirements.txt`? Perhaps name it `requirements-optional.txt`?

Reply:
Correted with `pip install -r requirements.txt`